Remastered Sound Effects

imlatefortheshow opened the pull request:
Replace the old sounds with better quality sounds

imlatefortheshow committed:
Remastered Classic SFX

Replaced the old sound effects with new, high quality versions.
In addition, SMB2 sounds were replaced with the ones from DDP.

imlatefortheshow committed:
Remastered Mario Announcer Voice

Let's a-go!

mmatyas commented:
Nice! @Ganbare-Lucifer could you take a look on these files?

As I'm far from being an expert in sound tech, could one of you explain the changes? Were these ripped from a more recent game (and if so, which one)?

imlatefortheshow commented:
Most high-quality sound effects I gathered were from the Super Mario World soundtrack album's second disc which contains music and sounds from Super Mario Bros., Super Mario Bros.: The Lost Levels, Super Mario Bros. 3 and Super Mario World.
I couldn't find a raw, higher quality version of the Bob-omb explosion sound (bob-omb.wav) from Super Mario All-Stars, so I had to use the castle explosion sound from Super Mario World instead.
The burning voice (burnup.wav) is a straight rip from Mario Kart 64.
The stored powerup sound (storedpowerup.wav) and the "Let's-a go!" voice (lets_go.wav) are straight rips from Super Mario 64.
I replaced the old chicken sound (chicken.wav) with a new similar-sounding one that's higher quality.
I replaced the sounds from Super Mario Bros. 2 (clock.wav, pickup.wav and throw.wav) with the ones from Doki Doki Panic.

Ganbare-Lucifer commented:
High Quality rips of SNES Mario games sound effects are found on LadiesMan's Ultimate All-Stars OST, he ripped and made brand new ones from direct hardware

Lemme see if I can grab them, here's it
https://drive.google.com/drive/folders/1HOmEtnideNE6Im7rjMKSJW5m_OR5HAJM

imlatefortheshow committed:
Enter Stage and Bob-omb sound effects

imlatefortheshow committed:
Enter Stage sound

mmatyas commented:
Ok, sounds good to me! I see there are more recent commits, let me know if this is ready to merge.

imlatefortheshow commented:
I think we're all done now. It's merging time!
